.github/workflows/mockE2E.yml

```diff
@@ -0,0 +1,33 @@
+name: Test e2e for PR
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  build-and-test-e2e-mock:
+    runs-on: ubuntu-20.04
+    name: Test e2e for PR using mock app
+    steps:
+      - uses: actions/checkout@v4
+      - name: 'Install pnpm'
+        uses: pnpm/action-setup@129abb77bf5884e578fcaf1f37628e41622cc371
+        with:
+          version: 8
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.node-version'
+      - name: Install dependencies
+        run: pnpm install
+      - name: Install Playwright
```

Comment:
Do we need to install playwright for mocked e2e tests?

Reply:
From other CI workflows, as far as I remember, even if Playwright is not used, it needs to be installed because other workflows were failing when Playwright was not installed. We need to refactor our CI to fix this.